aqua/qt53/Portfile

```diff
@@ -0,0 +1,1775 @@
+
+                # https://codereview.qt-project.org/#/c/172619/
+                # patchfiles-append patch-qtconn-for-10.12.diff
+
+                # see https://bugreports.qt.io/browse/QTBUG-62658
+                # patchfiles-append patch-qtconnectivity-sdk10_13.diff
+
+                # @property UUID was removed in SDK 10.13
+                # avoid osx/osxbtcentralmanager.mm:241:34: error: call to implicitly-deleted copy constructor of 'ObjCScopedPointer<NSMutableArray>'
+                # @property isConnected was removed in SDK 10.13
+                # patchfiles-append patch-qtconnectivity-uuid.diff
+            }
+
+            # special case
+            if { ${module} eq "qtwebkit-examples" && ![variant_isset examples] } {
+                # create a dummy file so the port can be successfully activated
+                post-destroot {
+                    xinstall -d -m 0755 ${destroot}${prefix}/share/doc/${subport}
+                    set docfile [open ${destroot}${prefix}/share/doc/${subport}/README.txt "w"]
+                    puts ${docfile} "Without the examples variant, no examples are installed"
+                    puts ${docfile} "${long_description}"
+                    close ${docfile}
+                }
+            }
+
+            # special case
+            if { ${module} eq "qtscript" } {
+                # see https://trac.macports.org/ticket/54453
+                patchfiles-append patch-qtscript_ceil.diff
+            }
+        }
+    }
+
+    if { [lsearch -exact [lindex ${module_info} 6] "~docs"] == -1 } {
+
+        subport ${name}-${module}-docs {
+
+            supported_archs   noarch
+            installs_libs     no
+            universal_variant no
+
+            distname        ${module}-opensource-src-${version}
+
+            if { ${revision_string} ne "0" } {
+                revision ${revision_string}
+            }
+
+            description       Documentation for the port ${name}-${module}
+            long_description  ${description}
+
+            build.target    docs
+            destroot.target install_docs
+
+            checksums                                        \
+                rmd160  [lindex [lindex ${module_info} 0] 0] \
+                sha256  [lindex [lindex ${module_info} 0] 1] \
+                size    [lindex [lindex ${module_info} 0] 2]
+
+            depends_build-append            \
+                port:${name}-qttools        \
+                port:${name}-sqlite-plugin
+
+            # depend on the modules for which this subport provides documentation
+            # also have the same build dependencies
+            depends_build-append port:${name}-${module}
+            foreach deps [lindex ${module_info} 1] {
+                depends_build-append [subst ${deps}]
+            }
+
+            PortGroup           qmake5 1.0
+
+            qt5.debug_variant   no
+
+            # special cases
+            if { ${module} eq "qttools" || ${module} eq "qtbase" } {
```

Comment:
I wonder why `|| ${module} eq "qtbase"` is not needed in later versions of Qt 5.
It is a low priority question.

Reply:
Weird one. Just trial-and-errored it. I guessed that the qtbase build assumed you were building qtbase with the docs, and qdoc would be there...